packages/perseus/src/widgets/__tests__/expression.test.tsx

```diff
@@ -180,7 +180,7 @@ describe("Expression Widget", function () {
         });
     });
 
-    describe("invalid", function () {
+    describe.skip("invalid", function () {
```

Comment:
Can we bring this back or is it flaky/broken in some way?

Reply:
I'll renable this before landing.